lib/create-issue.js

```diff
@@ -1,19 +1,20 @@
 module.exports = createIssue
 
 function createIssue (state) {
-  const {filename, patch, branchUrl} = state.commit
+  const {filename, patch, branchUrl, message, committer} = state.commit
   const content = state.template
     .replace(/\$DIFF/, patch)
     .replace(/\$FILENAME/, filename)
     .replace(/\$BRANCH_URL/, branchUrl)
     .replace(/\$REPO/, state.installRepo)
+    .replace(/\$COMMITTER/, `@${committer}`)
```

Comment:
can you please rename `$COMMITTER` to `$AUTHOR` and `committer` to `authorLogin`